### DISCUSSION TITLE
blog/2023-05-21-causal-inference-with-ordinal-regression/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#19] [Comments]
# blog/2023-05-21-causal-inference-with-ordinal-regression/

Part 7 of the GLM and causal inference series.

https://solomonkurz.netlify.app/blog/2023-05-21-causal-inference-with-ordinal-regression/

**Comment by joepax** [comment]
Hey Solomon. Why do we bother to extract the counterfactual epred values for each respondent? What benefit does that provide, vs. just extracting the epred values for the observed condition that each respondent was actually in? -Joe

**Reply by ASKurz** [reply]
Hey @joepax, I've only explored that option a little, and I believe the results are the same. The primary reason I've relied on the counterfactuals is because it makes the code simpler.

**Comment by DrJerryTAO** [comment]
You are right that the notation could be improved. Φ(pj) = tau_j looks wrong. It should be Φ^-1(p1 + ... + pj) = tau_j, and ANCOVA models should also use an inverse normal link instead of logit.

**Reply by ASKurz** [reply]
Thanks for the catch on the LateX!

**Comment by DrJerryTAO** [comment]
"I believe this is the first time we’ve had a model where baseline covariates increased the posterior SD for beta1 AND for the estimate of the causal estimand." The table above it does not show so. Instead, it shows that adding covariates increased the standard error bur reduced the point estimate, and this pattern also emerged in other types of models according to your other blogs. Nevertheless, it is not surprising, as coefficients between nested generalized linear models cannot be compared directly. See Williams, R., & Jorgensen, A. (2023). Comparing logit & probit coefficients between nested models. Social Science Research, 109, 102802. https://doi.org/10.1016/j.ssresearch.2022.102802

**Reply by ASKurz** [reply]
I think my meaning got obscured by some poor word choices. My focus is on the poster SDs, here, not the posterior mean (the analogue to the point estimate). Hopefully the updated version of the sentence makes this more clear.

**Comment by JackGuo15** [comment]
Hi Solomon,

Thank you so much for your super useful tutorial!! I'm wondering if it is also possible to calculate "average treatment effect for latent potential outcomes" for an adjacent-category model. I have tried using avg_comparison(), such as:
avg_comparisons(
  model,
  variables = list(predictor = "pairwise"),
  type = "link"  
)
But the output is grouped by the rating option, a bit similar to conditional treatment effect for each response option. I'm wondering if it is possible to calculate an overall effect, such as, compared with treatment B, is treatment A going to be higher or lower on the latent outcomes.

**Reply by ASKurz** [reply]
Hey @JackGuo15, thanks for your interest. Unfortunately, I don't have experience with adjacent category models, so I'm not in a good position to comment. This might be a question for the Stan forums, or the {marginaleffects} GitHub repo.

**Reply by JackGuo15** [reply]
Hi Solomon,

Thank you so much for your reply! I think I successfully did this by getting the pairwise contrasts using avg_comparisons(), and then extract posterior draws (using posterior_draws) and average across categories. I will double check if this is correct on the Stan at some points. Thanks again for your answer and this amazing tutorial! 
Have a wonderful day!

Best wishes,
Jack
